Source: Upstream
Upstream: Yes
Reason: Fix missing shared library versioning

From 00819877651b87842ed878898ba17dba489820f0 Mon Sep 17 00:00:00 2001
From: Daniel Veillard <veillard@redhat.com>
Date: Thu, 4 Nov 2010 21:53:14 +0100
Subject: [PATCH] Reactivate the shared library versionning script

---
 configure.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/configure.in b/configure.in
index 59d0629..a1d2c89 100644
--- a/configure.in
+++ b/configure.in
@@ -84,7 +84,7 @@ else
   esac
 fi
 AC_SUBST(VERSION_SCRIPT_FLAGS)
-AM_CONDITIONAL([USE_VERSION_SCRIPT], [test -z "$VERSION_SCRIPT_FLAGS"])
+AM_CONDITIONAL([USE_VERSION_SCRIPT], [test -n "$VERSION_SCRIPT_FLAGS"])
 
 dnl
 dnl We process the AC_ARG_WITH first so that later we can modify
-- 
1.7.3.2

